Terra/Workspaces/BloodPressure_GWAS_Template.md

```diff
@@ -90,26 +82,26 @@ Gen3 uploaded tar compressed bundles, as they are provided by dbGAP, into cloud
-10. Generate a genetic relatedness matrix ([GRM](https://hail.is/docs/0.2/methods/genetics.html?highlight=pc_rel#hail.methods.genetic_relatedness_matrix)) to account for closely related individuals in your association testing workflows.
-11. Generate a new "sample_set" data table that holds the derived files we created in the steps above using the [FireCloud Service Selector (FISS) package](https://github.com/broadinstitute/fiss).  The files in this data table will be used in the workflows we run in Part 3.     
-
-
+2. Import the multi-sample VCF from the "Reference File" data table using DRS. You can learn more about GA4GH's Data Repository Service [here](https://support.terra.bio/hc/en-us/articles/360039330211).
+3. Filter your VCF to only common variants to increase statistical power. Genetic analyses in this notebook utilize the [Hail software](https://hail.is/). Hail is a framework for distributed computing with a focus on genetics. Particularly relevant for whole genome sequence ([WGS](https://en.wikipedia.org/wiki/Whole_genome_sequencing)) analysis, Hail allows for efficient, nearly boundless computing (in terms of variant and sample size).    
+4. Perform a principal component analysis ([PCA](https://en.wikipedia.org/wiki/Principal_component_analysis)) to assess population stratification. Genetic stratification can strongly affect association tests and should be accounted for.
+5. Generate a genetic relatedness matrix ([GRM](https://hail.is/docs/0.2/methods/genetics.html?highlight=pc_rel#hail.methods.genetic_relatedness_matrix)) to account for closely related individuals in your association testing workflows.
+6. Generate a new "sample_set" data table that holds the derived files we created in the steps above using the [FireCloud Service Selector (FISS) package](https://github.com/broadinstitute/fiss).  The files in this data table will be used in the workflows we run in Part 3.     
 
 ### Time and cost estimate
-You can adjust the runtime configuration to fit your computational needs in the Jupyter notebook. We recommend selecting the default environment and selecting the custom profile to use and configure the spark cluster for parallel processing.  Using the profile suggested profile within the Jupyter notebook and a project with around 1000 samples, running this notebook on this dataset takes about 90 minutes and $20/hr to compute.
- 
+You can adjust the runtime configuration to fit your computational needs in the Jupyter notebook. We suggest using the recommended computes as explained in each notebook. The third notebook will be your most expensive. Terra will provided a per-hour estimate of cost as you adjust settings of your compute environment. The first two notebooks can usually be completed in about 30 minutes total, or less. Be aware that if you are running on more than 1000 full-sized samples, the third notebook may take upwards of an hour to complete.
```

Comment:
I would replace line 104 with something like: "The first two notebooks should run with the default environment and runtime configuration. The third notebook *3-should use the latest Hail image and the associated runtime environment should use the Spark cluster. However, the number of workers, CPUs, and memory will vary drastically based on your input files and cost/time needs. We provide some rule of thumb examples for you to consider for your Spark cluster configuration."

And then I would add an explanation of the relationship of worker and cpu:core. And that the Hail team recommends this general rule of thumb:
- 128 MB per partition
- 3:1::partitions:cores